src/io/flutter/preview/PreviewView.java

```diff
@@ -357,6 +359,11 @@ public void invokePopup(Component comp, int x, int y) {
           hasAction = true;
           group.add(actionRemove);
         }
+        group.addSeparator();
+        if (actionExtractMethod.isEnabled()) {
```

Comment:
If we change the location in the toolbar where this is contributed, we should update here as well.